tests/pyfunc/test_spark.py

```diff
@@ -140,7 +135,7 @@ def predict(self, context, model_input):
                 columns=["a", "b", "c", "d"], data={"a": [1], "b": [2], "c": [3], "d": [4]}
             )
         )
-        with pytest.raises(Py4JJavaError):
+        with pytest.raises(pyspark.sql.utils.PythonException):
```

Comment:
Fix for this failure:

https://github.com/mlflow/mlflow/runs/2330220978#step:6:611